core/src/main/scala/io/finch/Bootstrap.scala

```diff
@@ -123,7 +123,7 @@ class Bootstrap[F[_], ES <: HList, CTS <: HList](
 
   def toService(implicit F: Async[F], ts: Compile[F, ES, CTS]): Resource[F, Service[Request, Response]] = {
     val compiled = compile
-    Dispatcher[F].flatMap { dispatcher =>
+    Dispatcher.parallel[F].flatMap { dispatcher =>
```

Comment:
This was the default behavior prior 3.4.0, but I wonder if we should not make this to be parametrable.